CHANGELOG.md

```diff
@@ -6,7 +6,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
```

Comment:
We should also document the whole change to procmacro :D

Reply:
I planned to leave that to the other PR: #36.